systemvm/image/scripts/install_systemvm_packages.sh

```diff
@@ -96,10 +90,9 @@ function install_packages() {
   # Install docker and containerd for CKS
   curl -fsSL https://download.docker.com/linux/debian/gpg | sudo apt-key add -
   apt-key fingerprint 0EBFCD88
-  add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/debian $(lsb_release -cs) stable"
+  echo "deb [arch=amd64] https://download.docker.com/linux/debian bullseye stable" > /etc/apt/sources.list.d/docker.list
   apt-get update
   ${apt_get} install containerd.io
-
   apt_clean
```

Comment:
Hi @rohityadavcloud @Pearl1594 the script fails for me on Ubuntu 20.04 at this line:

````
/scripts/install_systemvm_packages.sh: line 96: apt_clean: command not found
+ fail 'install_systemvm_packages.sh failed'
+ clean_env
+ '[' /tmp/tmp.x7wCjvUAzz '!=' '' ']'
+ chroot /tmp/tmp.x7wCjvUAzz umount /proc /sys /dev /boot
+ '[' /tmp/tmp.x7wCjvUAzz '!=' '' ']'
+ umount /tmp/tmp.x7wCjvUAzz
+ sync
+ '[' /dev/nbd0 '!=' '' ']'
+ qemu-nbd -d /dev/nbd0
/dev/nbd0 disconnected
+ sync
+ '[' /tmp/tmp.x7wCjvUAzz '!=' '' ']'
+ rm -fr /tmp/tmp.x7wCjvUAzz
+ echo 'Failed: install_systemvm_packages.sh failed'
Failed: install_systemvm_packages.sh failed
+ exit 1
````

Reply:
After removing line 96 it suceeds (prepended time command):
````
SystemVM image generated: image.qcow2

real	5m37.860s
user	2m48.715s
sys	1m6.678s
````

Reply:
Thanks @nvazquez for testing; indeed the build time looks fair (compared to packer which takes about 20-40mins)